lib/github.ml

```diff
@@ -137,3 +139,19 @@ let gh_link_of_string url_str =
       with _ -> None
     end
   | _ | (exception Re2.Exceptions.Regex_match_failed _) -> None
+
+let get_project_owners (labels : label list) project_owners_map =
+  List.fold_left labels ~init:[] ~f:(fun acc label ->
+    match Map.find_multi project_owners_map label.name with
+    | [] -> acc
+    | reviewers -> List.rev_append reviewers acc
+  )
+  |> List.dedup_and_sort ~compare:String.compare
+  |> List.partition_map ~f:(fun reviewer ->
+       try
+         let team = Re2.find_first_exn ~sub:(`Index 1) gh_org_team_re reviewer in
+         Second team
+       with Re2.Exceptions.Regex_match_failed _ -> First reviewer
```

Comment:
Would it make sense to make the error visible? e.g. by posting a message on the pull request.

If I understand correctly, showing these errors prominently would help keep the PROJECTOWNERS file up to date when members leave an org, etc.